tasks/prReviewReminder.ts

```diff
@@ -0,0 +1,42 @@
+import { danger } from "danger"
+
+export interface PRReviewMetadata {
+  repoName: string
+  prNumber: number
+  requestedReviewer: string
+  owner: string
+}
+
+export default async (metadata: PRReviewMetadata) => {
+  const pullParams = {
+    owner: metadata.owner,
+    repo: metadata.repoName,
+    number: metadata.prNumber,
+  }
+  // Get the PR we want to add a comment to
+  const pr = await danger.github.api.pulls.get(pullParams)
+  const reviews = await danger.github.api.pulls.listReviews(pullParams)
+  const currentReviewers: string[] = pr.data.requested_reviewers.map(user => user.login)
+
+  // Confirm that the PR is still open and the initially requested reviewer is still requested
+  if (pr.data.state === "open" && currentReviewers.includes(metadata.requestedReviewer)) {
+    // Loop through the reviews and see if this reviewer has already reviewed
+    for (let i = 0; i < reviews.data.length; i++) {
+      // If they have, just return
+      if (reviews.data[i].user.login === metadata.requestedReviewer) {
+        return
+      }
+      // If we've looped through all the reviews and didn't find one by our reviewer,
```

Comment:
If I'm reading this correctly, don't we want to put this comment and the following code outside the scope of the `for` loop? Otherwise we could get into some extra comments getting made erroneously.

Reply:
oh lordy you are indeed correct. addressed in the most recent commit